libs/sdk-ui-kit/src/Dialog/StylingEditorDialog/ColorPreview.tsx

```diff
@@ -0,0 +1,28 @@
+// (C) 2022 GoodData Corporation
+import cx from "classnames";
+import React from "react";
+
+/**
+ * @internal
+ */
+export interface IColorPreviewProps {
+    colors: string[];
+    className?: string;
+}
+
+/**
+ * @internal
+ */
+export const ColorPreview = (props: IColorPreviewProps) => {
```

Comment:
can this component be used also in StylingPickerListItem.tsx to replace almost the same code?
{colorsPreview.map((color, index) => (
                        <span
                            key={index}
                            className="gd-styling-picker-list-item-color"
                            style={{ backgroundColor: color }}
                        />
                    ))}